.storybook/helpers/files.data.ts

```diff
@@ -50,15 +72,30 @@ export const filesRoot: File[] = [
     ],
   },
   {
+    disabled: true,
+    selected: ClrSelectedState.UNSELECTED,
     name: 'package.json',
   },
   {
     name: 'tsconfig.json',
   },
 ];
 
-export function getFileTreeNodeMarkup(files: File[] = filesRoot) {
+export function getFileTreeNodeMarkup(files: File[] = filesRoot, clrSelected?: boolean) {
   return files
-    .map(file => `<clr-tree-node>${file.name}${file.files ? getFileTreeNodeMarkup(file.files) : ''}</clr-tree-node>`)
+    .map(file => {
+      let selected = '';
+      if (clrSelected !== undefined && file.selected !== undefined) {
+        selected = `[clrSelected]="${file.selected}"`;
+      }
```

Comment:
```suggestion
      const selected = clrSelected !== undefined && file.selected !== undefined ? `[clrSelected]="${file.selected}"` : '';
```

Reply:
Entire selected functionality is removed, because make no sense in storybook anymore.